src/metabase/query_processor.clj

```diff
@@ -176,7 +176,7 @@
   it. This only works for pure MBQL queries, since it does not actually run the queries. Native queries or MBQL
   queries with native source queries won't work, since we don't need the results."
   [{query-type :type, :as query}]
-  (when-not (= query-type :query)
+  (when-not (= (keyword query-type) :query)
```

Comment:
perhaps `mbql.u/normalize-token` is a more robust choice?

Reply:
ahhh I didn't even know about that.